### DISCUSSION TITLE
Deep Sleep

### DISCUSSION THREAD

**Discussion by sikorskimarcin** [RavenSystem/esp-homekit-devices#1766] [Q&A] [answered]
Hi guys, 
I was searching at the whole repository but I can't find any info about Deep Sleep capabilities. 
That would be great functionality for thermometers and hygrometers and maybe for push buttons. 

Saludos,

Marcin

**Comment by sikorskimarcin** [comment]
@RavenSystem ¿puedes ayudar?

**Comment by RavenSystem** [comment, marked as answer]
Deep sleep is incompatible with WiFi HomeKit devices. You must Bluetooth LE.

**Reply by sikorskimarcin** [reply]
Thank you. Muchas gracias.
